Kickstarter-iOS/Views/Controllers/DiscoveryViewController.swift

```diff
@@ -175,3 +175,19 @@ extension DiscoveryViewController: DiscoveryNavigationHeaderViewDelegate {
     self.filter(with: params)
   }
 }
+
+extension DiscoveryViewController: TabBarControllerScrollable {
+  func scrollToTop() {
```

Comment:
Unfortunately due to the nature and setup of `DiscoveryViewController` I didn't find a good way to test this.